app/assets/javascripts/discourse/app/services/element-classes.js

```diff
@@ -3,36 +3,36 @@ import Service from "@ember/service";
 import { disableImplicitInjections } from "discourse/lib/implicit-injections";
 
 @disableImplicitInjections
-export default class BodyClassesService extends Service {
-  #helpers = new Map();
+export default class ElementClassesService extends Service {
+  #helpers = new Map(); // Map<Helper, string[]>
```

Comment:
```suggestion
  /** @type Map<helper, string[]> */
  #helpers = new Map();
```
^^ notating this way makes it understandable to IDEs

<img width="491" alt="SCR-20231204-kagg" src="https://github.com/discourse/discourse/assets/6270921/7d7059d9-59cb-4df4-95ca-fe64b161559f">